cookbook/knowledge/basic_knowledge_manager.py

```diff
@@ -12,10 +12,10 @@
 knowledge = Knowledge(
     name="Basic SDK Knowledge Base",
     description="Agno 2.0 Knowledge Implementation",
-    vector_store=PgVector(
+    vector_db=PgVector(
```

Comment:
I like vector store, because some of the technologies is more like a store than a DB... so vector store seems more general?

Or wdyt?